website/screens/overview/introduction/IntroductionPage.tsx

```diff
@@ -185,6 +185,47 @@ const sections = [
       </>
     ),
   },
+  {
+    title: "Documentation of other releases",
+    content: (
+      <>
+        <DxcParagraph>
+          This site is new and comes to replace the existing one. However, the
+          old documentation is still available through the following links for
+          consultation:
+        </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            <DxcLink href="https://developer.dxc.com/tools/react" newWindow>
+              React CDK
+            </DxcLink>
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <DxcLink href="https://developer.dxc.com/tools/angular" newWindow>
+              Angular CDK
+            </DxcLink>
+          </DxcBulletedList.Item>
+        </DxcBulletedList>
+        <DxcParagraph>
+          <em>
+            Please note that these sites are discontinued and may not be updated
+            anymore.
+          </em>
+        </DxcParagraph>
+        <DxcParagraph>
+          The last development site can be accessed through this{" "}
+          <DxcLink href="https://developer.dxc.com/halstack/next/" newWindow>
+            link
+          </DxcLink>
+          ; and the last release is available{" "}
+          <DxcLink href="https://developer.dxc.com/halstack/6/" newWindow>
+            here
```

Comment:
There is no way to take the last release programatically instead of change it manually?

Reply:
Good idea! I think we can take it from the env variable.

Reply:
Done!